LondonBritishLibrary/orient/BLorient786.xml

```diff
@@ -0,0 +1,239 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient786" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Rǝtuʿa Haymānot, Collection of homilies for various occasions</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>Wright cat. CCCXLIII</idno>
+                    </msIdentifier>
+
+<msContents>
+
+
+  <msItem xml:id="ms_i1">
+  <locus from="5r"/>
+  <title type="complete" ref="LIT2222RetuaH">Rǝtuʿa Hāymānōt, or "the Orthodox", a collection of homilies for various occasions</title>
+  <textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.1">
+<locus from="5r"/>
+<title type="complete">Introduction</title>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="10v"/>
+<title type="complete" ref="LIT2113OntheB" xml:lang="en">On the birth of our Lord of the blessed Virgin Mary</title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#10v" xml:lang="gez"></locus>ድርሳን፡ ዘርቱዐ፡ ሃይማኖት፡ በእንተ፡ ልደተ፡ አምላክ፡ እማርያም፡ እምቅድስት፡</incipit>
+</msItem>
+
+
+<msItem xml:id="ms_i1.3">
+<locus from="24v"/>
+<title type="complete" ref="LIT2095OnHowO">On the baptism of our Lord</title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#24v" xml:lang="gez"></locus>በእንተ፡ ዘከመ፡ ተጠምቀ፡ አምላክነ፡ ክርስቶስ፡ በፈለገ፡ ዮርዳኖስ፡ በዕደ፡ ዮሐንስ፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="40r"/>
+<title>For the beginning of Lent </title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#40r" xml:lang="gez"></locus>ዘይትነበብ፡ በቀበላ፡ ጾም፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="47r"/>
+<title type="complete" ref="LIT6049SundayRetuaH">For the (first) Sunday of Lent </title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#47r" xml:lang="gez"></locus>ድርሳን፡ ዘይትነበብ፡ በቅድስት፡ ጾም፡ ሰንበተ፡ ክርስቲያን፡ በእንተ፡ ጾም፡ እመ፡ ኵሉ፡ ምግባራት፡</incipit>
+</msItem>
```

Comment:
Same notes as in the previous ms_items, xml:lang for title and incipit. Same for further ms_items